src/azure-cli/azure/cli/command_modules/relay/custom.py

```diff
@@ -104,12 +104,12 @@ def cli_wcfrelay_update(instance, relay_type=None, user_metadata=None, status=No
 # Hybrid Connection Region
 def cli_hyco_create(client, resource_group_name, namespace_name, hybrid_connection_name,
                     requires_client_authorization=None, user_metadata=None):
-
     return client.create_or_update(
         resource_group_name=resource_group_name,
         namespace_name=namespace_name,
         hybrid_connection_name=hybrid_connection_name,
-        requires_client_authorization=requires_client_authorization, user_metadata=user_metadata)
+        requires_client_authorization=requires_client_authorization,
+        user_metadata=user_metadata)
```

Comment:
I would suggest you back out the cosmetic change if the command module is owned by other teams.

Reply:
we made these changes since this command was not working without this fix - 

az relay hyco create -g <RG> --namespace-name<servicebus> --name <app> --user-metadata  "[{\"key\":\"endpoint\",\"value\":\"something:80\"}]" we can add them to the PR if needed.

Reply:
but the change here is just a line breaks, no?

Reply:
Ah, looks like with the latest change the service bus folks have fixed the issue as well. So yes, we can reset this file to the upstream dev branch.